compose.yaml

```diff
@@ -5,15 +5,32 @@ services:
       context: .
       dockerfile: Dockerfile
     ports:
-      - "80:3000"
+      - "3000:3000"
     environment:
       - NODE_ENV=production
-      - PORT=3000
+      - PORT=${PORT}
       - HOST=${HOST}
       - GOOGLE_CLIENT_ID=${GOOGLE_CLIENT_ID}
       - GOOGLE_CLIENT_SECRET=${GOOGLE_CLIENT_SECRET}
       - COOKIE_SECRET=${COOKIE_SECRET}
       - INPUT_FILE_PATH=/input.json
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_HOST=${DB_HOST}
     volumes:
       - ./input.json:/input.json
-      - ./.db/:/usr/src/app/.db/
\ No newline at end of file
+      - ./.db/:/usr/src/app/.db/
+      - shared:/usr/src/app/public/cdoc_files
```

Comment:
Should cdoc files be available to everyone from a security perspective?

Reply:
CDOC files are already encrypted. Nothing to hide